reconcile/utils/ocm.py

```diff
@@ -579,6 +582,28 @@ def delete_machine_pool(self, cluster, spec):
             f'{machine_pool_id}'
         self._delete(api)
 
+    def get_available_upgrades(self, version, channel):
+        """Get available versions to upgrade from specified version
+        in the specified channel
+
+        Args:
+            version (string): OpenShift version ID
+            channel (string): Upgrade channel
+
+        Raises:
+            KeyError: if specified channel is not valid
+
+        Returns:
+            list: available versions to upgrade to
```

Comment:
Does this return the versions in a particular order (ascending numerically, descending numerically)?

Reply:
it returns it just as the OCM API returns it